packages/f-searchbox/src/components/Form.vue

```diff
@@ -153,7 +152,11 @@ export default {
          * @returns {*}
          */
         errorMessage () {
-            const messageKey = this.errors.length && this.errors[0];
+            const messageKey =
+                    this.isDirty
+                    && this.errors
```

Comment:
Could this not be cleaned up using optional chaining? E.g.
```suggestion
                    && this.errors?.[0];
```

Reply:
Yup ill update this PR when i return next week 👍

Reply:
Fixed.